src/components/views/login/PasswordLogin.js

```diff
@@ -56,33 +57,64 @@ class PasswordLogin extends React.Component {
         this.onPhoneCountryChanged = this.onPhoneCountryChanged.bind(this);
         this.onPhoneNumberChanged = this.onPhoneNumberChanged.bind(this);
         this.onPasswordChanged = this.onPasswordChanged.bind(this);
+        this.isLoginEmpty = this.isLoginEmpty.bind(this);
```

Comment:
ftr, I don't think you need to bind this as you're not passing it in as a callback anywhere. Arguably it's clearer if we just always bind component methods though.